Makefile

```diff
@@ -83,6 +83,10 @@ release:
 	docker tag $(CONTROLLER_IMAGE) $(REGISTRY)/$(CONTROLLER_IMAGE):latest
 	docker push $(REGISTRY)/$(CONTROLLER_IMAGE):latest
 
+manifests/golden-pvc.yaml: manifests/golden-pvc.yaml.in
```

Comment:
can you move this to the end?  The ugly, commented out code block blow is _eventually_ supposed to be part of `release:` :)